lib/security/allowed_github_ips.json

```
@@ -0,0 +1,4999 @@
+{
+  "GitHubAllowIpIpv4": [
```

We already have file named with allowed_github_ips.
```suggestion
  "ipv4": [
```

---

Updated.